app/controllers/courses_controller.rb

```diff
@@ -70,6 +70,22 @@ def show_json
     render json: data.to_json
   end
 
+  # This is for analysis and better points from exercise program, which will be done one day.
+  def submissions_data
```

Comment:
Name can be too easily confused with [submission_data.rb](https://github.com/testmycode/tmc-server/blob/master/app/models/submission_data.rb)